templates/_layout.html

```diff
@@ -9,17 +9,17 @@
 </head>
 <body>
     <nav class="navbar navbar-expand-lg navbar-light custom-navbar navbar-custom">
-        <a class="navbar-brand">PlayNest</a>
+        <a class="navbar-brand" href="{{ url_for('index') }}">PlayNest</a>
         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
             <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarNav">
             <ul class="navbar-nav">
                 <li class="nav-item active">
-                    <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+                    <a class="nav-link" href="{{ url_for('home') }}">Home <span class="sr-only">(current)</span></a>
                 </li>
                 <li class="nav-item">
-                    <a class="nav-link" href="#">Forum</a>
+                    <a class="nav-link" href="{{ url_for('forum') }}">Forums</a>
                 </li>
                 <li class="nav-item">
                     <a class="nav-link" href="#">Browse Games</a>
```

Comment:
Now that we have the dropdown menu implemented below, we can probably remove the "Browse Games" button on the nav bar.